data/interfaces.txt

```diff
@@ -672,6 +672,10 @@ interface "hud"
 	anchor top
 	point "mini-map"
 		center 0 100
+	
+	value "min zoom" .25
+	value "zoom multiplier" 1.4
+	value "max zoom" 2.
```

Comment:
```suggestion
	# The minimum (largest) and maximum (closest) zooms.
	# A multiplier between 0 and 1 will invert the direction of zoom.
	value "min zoom" .25
	value "zoom multiplier" 1.4
	value "max zoom" 2.
```

As discussed, some general comments so people can understand what this is and how to use it.

Reply:
Can you re-arrange this a bit whilst we're at it?
Like first comment for min max, then values for min max, then comment for multiplier, then multiplier